currencies/src/main/java/com/generalbytes/batm/common/currencies/Erc20Token.java

```diff
@@ -5,8 +5,8 @@
 import java.util.Map;
 
 public enum Erc20Token {
-    USDT(CryptoCurrency.USDT, "0xdAC17F958D2ee523a2206206994597C13D831ec7"),
-    DAI(CryptoCurrency.DAI, "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359"), // SAI stable coin
+    USDT(CryptoCurrency.USDT, 6, "0xdAC17F958D2ee523a2206206994597C13D831ec7"),
+    DAI(CryptoCurrency.DAI, 18, "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359"), // SAI stable coin
```

Comment:
Should the comment say "DAI stable coin"?

Reply:
> What Is SAI?
SAI is Single Collateral DAI. It is the non-migrated version of DAI. With the launch of Multi-Collateral DAI on November 18th, 2019, the name and ticker symbol of Single Collateral DAI is changing to SAI (SAI).

> What is DAI?
DAI is the new DAI. It is often referred to as Multi-Collateral DAI. It is an ERC20 token on the Ethereum blockchain that has aims to have a steady value of 1 USD. DAI has remained stable for several years with only minor wavers from its 1 USD price point.

The contract address we use is for the SAI from when it was still called DAI. It was used on POS only